### DISCUSSION TITLE
New list added problem. From mobile git hub app for android

### DISCUSSION THREAD

**Discussion by naidenpetrov00** [community/community#52273] [Other Feature Feedback, Questions, & Ideas]
When creating new list the pop up with suggested list is not refreshing and you have to close and open again to see it

**Comment by IldarKhayrutdinov** [comment]
I have the same problem on brave browser. When adding a repo to a new list, the creation of the list hangs. Wherein the list is created and the repository is added to the list.

**Reply by Gogolpop** [reply]
Всё отлично работает
